### DISCUSSION TITLE
How to change from static IP to DHCP?

### DISCUSSION THREAD

**Discussion by Sum-Guy** [arendst/Tasmota#17945] [Q&A] [answered]
If I have tasmota set to static IP, how do I change it back to dynamic or DHCP?  I presume this would be a console ipaddress1 command of some sort?  Does this also clear the other ipaddress settings?

**Comment by Jason2866** [comment, marked as answer]
set `ipaddress<x>` to `0.0.0.0` for DHCP

**Reply by Sum-Guy** [reply]
ok, and then it looks like I have to issue a restart for it to take effect.
